### PR TITLE
#157801293 Test for inviting users

### PR DESCRIPTION
**Pull request by Rhotimee**
#### What does this PR do?
Write test for inviting users

#### Description of Task to be completed?
Test was written for inviting users
Test was written for searching users

#### How should this be manually tested?
`npm test`

#### What are the relevant pivotal tracker stories?
157801293

### CODE REVIEW

File: test/endpoints/invite.spec.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const request = supertest.agent(app);
+
+describe('POST to /invite', () => {
+  it('Should send an invite successfully', (done) => {
+    request
+      .post('/api/invite')
+      .send({
+        gameURL: 'localhost:3000',
+        recieverEmail: 'douglas@gmail.com',
+        token: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjk1Mjg5MzIxMzEsIl9pZCI6IjViMjA1NWEzNDc1NGFhMDVmYjBlZjc5OSIsImVtYWlsIjoiZG91Z2xhc2VnaWVtZWhAZ21haWwuY29tIiwiaWF0IjoxNTI4ODQ1NzMxfQ.gOaq1u2hgcz4Hv_HJ8p0knOPIuXe_9PkoRqW0oJXx8g'
```
Comment (fortunee) [inline test/endpoints/invite.spec.js]: I doubt your test will pass when this token expires. What better ways can we simulate this? @Rhotimee

File: test/endpoints/invite.spec.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      });
+  });
+});
+
+describe('get to /search', () => {
+  it('Should search users', (done) => {
+    request
+      .post('/api/search')
+      .send({
+        term: '',
```
Comment (fortunee) [inline test/endpoints/invite.spec.js]: What user are we searching for? @Rhotimee

**Review by fortunee — COMMENTED** [review]
Hi, @Rhotimee I'll be willing to discuss my comment on this PR over a call and other ways we can improve this test cases. 😄

File: test/endpoints/invite.spec.js (hunk 2 above)
Comment (Rhotimee) [reply test/endpoints/invite.spec.js]: No user in particular, If a test for signup has been written, i can then search for the signedup user.

**Pull request closed without merging**